.github/files/setup-wordpress-env.sh

```diff
@@ -133,6 +133,43 @@ for PLUGIN in projects/plugins/*/composer.json; do
 	echo "::endgroup::"
 done
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+# Todo: check the require paths being used by the Woo tests, they may need updating since the Woo repo structure changed.
+# Todo: what to check so this only runs when Jetpack will be tested?
```

Comment:
First you'd want to update https://github.com/Automattic/jetpack/blob/38d1aa2ea475761eabb98666329f1feeaadd94ad/.github/workflows/tests.yml#L87-L89 to add
```yaml
        env:
          CHANGED: ${{ steps.changed.outputs.projects }}
```
to pass the data into the script via the environment.

Then you could do like this to check for Jetpack-the-plugin
```sh
if [[ "$WP_BRANCH" == "latest" && "$PHP_VERSION" == "8.2" ]] &&
    jq --argjson changed "$CHANGED" -ne '$changed["plugins/jetpack"] // false' > /dev/null
then
```

Reply:
Awesome, be91bef